### PR TITLE
USHIFT-568: Apply fixes to code and docs to run on aarch64 platform

### PR DESCRIPTION
**Pull request by ggiguash**
- Fix docs to remove hard-coded x86_64 mentions and add aarch64 support on RHEL 8.7 OS
- Minor code fixes to error message formatting
- Add aarch64 container image references (all except TopoLVM should work)
- Fix partitioning to split /boot into 200/800MB for efi

Closes [USHIFT-568](https://issues.redhat.com//browse/USHIFT-568)

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]
/lgtm

File: pkg/release/release_arm64.go [hunk 1]
```diff
@@ -25,9 +25,14 @@ func init() {
...
+		"odf_topolvm":               "registry.redhat.io/odf4/odf-topolvm-rhel8@sha256:362c41177d086fc7c8d4fa4ac3bbedb18b1902e950feead9219ea59d1ad0e7ad",
+		"openssl":                   "registry.access.redhat.com/ubi8/openssl@sha256:8b41865d30b7947de68a9c1747616bce4efab4f60f68f8b7016cd84d7708af6b",
+		"ose_csi_ext_provisioner":   "registry.redhat.io/openshift4/ose-csi-external-provisioner@sha256:4b7d8035055a867b14265495bd2787db608b9ff39ed4e6f65ff24488a2e488d2",
+		"ose_csi_ext_resizer":       "registry.redhat.io/openshift4/ose-csi-external-resizer@sha256:ca34c46c4a4c1a4462b8aa89d1dbb5427114da098517954895ff797146392898",
+		"ose_csi_node_registrar":    "registry.redhat.io/openshift4/ose-csi-node-driver-registrar@sha256:3babcf219371017d92f8bc3301de6c63681fcfaa8c344ec7891c8e84f31420eb",
+		"ose_csi_livenessprobe":     "registry.redhat.io/openshift4/ose-csi-livenessprobe@sha256:e4b0f6c89a12d26babdc2feae7d13d3f281ac4d38c24614c13c230b4a29ec56e",
+		"ovn_kubernetes_microshift": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2fdcafe1efb12e28b2ea30823633c097d8990840ad66f2c6025969281ea80316",
```
Comment (fzdarsky) [inline pkg/release/release_arm64.go:34]: Why did this image hash change? It should only be changed by the rebase automation.
Same as for the "pod" image.

Comment (ggiguash) [reply pkg/release/release_arm64.go:34]: I updated the images with rc0 hashes. The next rebase should fix it and make it all consistent, right?

**Pull request merged**